DREAM3DReviewFilters/FindArrayStatistics.cpp

```diff
@@ -434,6 +462,74 @@ double findSummation(C<T, Ts...>& source)
   return sum;
 }
 
+// -----------------------------------------------------------------------------
+//
+// -----------------------------------------------------------------------------
+template <template <typename, typename...> class C, typename T, typename... Ts>
+std::vector<float> findHistogram(C<T, Ts...>& source, float histmin, float histmax, bool histfullrange, int32_t numBins)
+{
+
+  std::vector<float> Histogram(numBins, 0);
+  int overflow = 0;
+  int32_t bin = 0;
+  int32_t numPoints = source.size();
+
+  if(source.empty())
+  {
+    std::vector<float> empty(numBins, 0);
+    return empty;
+  }
+
+  float min = std::numeric_limits<float>::max();
+  float max = -1.0 * std::numeric_limits<float>::max();
+
+  if(histfullrange)
+  {
+    min = findMin(source);
+    max = findMax(source);
+  }
+  else
+  {
+    min = histmin;
+    max = histmax;
+  }
+
+  float increment = (max - min) / (numBins);
+  if(abs(increment) < 1E-10)
```

Comment:
```suggestion
  if(std::abs(increment) < 1E-10)
```
C standard library functions should be prefixed with std::